spec/car_spec.rb

```diff
@@ -37,4 +37,9 @@
 
     expect(car.total_cost).to eq(54000)
   end
+
+  it 'has a color' do
+    car = Car.new("Ford Mustang", 1500, 36)
+    expect(car.paint!(:blue)).to eq(car.color)
```

Comment:
You would need to add another test that tests that it can get painted:
```
car.paint!(:blue)

it "can get paint to change the color" do
    car = Car.new("Ford Mustang", 1500, 36)
    car.paint!(:mint_green)
    expect(car.color).to eq(:mint_green)
  end
```